test/cid-version-agnostic.js

```diff
@@ -32,11 +32,17 @@ describe('CID version agnostic', function () {
     Object.assign(daemons, { js0, js1, go0, go1 })
 
     await Promise.all([
+      // @ts-ignore
```

Comment:
It's generally better to use `@ts-expect-error` as we usually add an ignore as a temporary measure but they have a way of sticking around forever.  `expect-error` will start to fail if the underlying issue is fixed.

That said, what is the underlying issue that needed all these ignores adding?

Reply:
Is it because this PR needs merging? https://github.com/ipfs/js-kubo-rpc-client/pull/25

Reply:
yea let me look at that PR